draft-ietf-quic-qpack.md

```diff
@@ -1551,6 +1551,128 @@ the smallest number of bytes.
 | 97    | x-frame-options                  | deny                                                        |
 | 98    | x-frame-options                  | sameorigin                                                  |
 
+
+# Encoding and Decoding Examples
+
+The following examples represent a series of exchanges between an encoder and a
```

Comment:
It would be good to give a high-level description of what exactly this series of exchanges shows/is intended to show up-front.